tests/fast-integration/eventing-test/common/common.js

```diff
@@ -0,0 +1,23 @@
+const fs = require('fs');
+const eventMeshSecretFilePath = process.env.EVENTMESH_SECRET_FILE || '';
+const eventMeshNamespace = getEventMeshNamespace();
```

Comment:
(nit) Can we only call `getEventMeshNamespace()` if `eventMeshSecretFilePath` is not empty? Otherwise, we would have error log in console every time we run tests for nats.

Reply:
I just moved the code here to fix the circular dependency.
Also this situation is not going to happen because this condition here: https://github.com/marcobebway/kyma/blob/ea8923a743c02dde991baebb5c0757e1012e5044/tests/fast-integration/eventing-test/eventing-test.js#L127.